client/build.rs

```diff
@@ -114,7 +114,7 @@ fn build_smartcontract(smartcontract_path: &Path, out_dir: &OsStr) -> Result<()>
         .env_remove("CARGO_ENCODED_RUSTFLAGS")
         .env("CARGO_TARGET_DIR", out_dir)
         .current_dir(smartcontract_path)
-        .args(&[
+        .args([
             "+nightly-2022-08-15",
```

Comment:
is this a good time to bump the version? We haven't bumped it in a while and we'd finally get stable GATs